.github/ISSUE_TEMPLATE/bug-performance-report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug/Performance report
+about: Create a report to help us improve
+title: "[Bug/Performance]"
```

Comment:
Can we remove `title`, `labels` and `assignees`? We can add it later if necessary

Reply:
removed